src/kernel/chainparams.cpp

```diff
@@ -340,8 +340,8 @@ class CTestNet4Params : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].min_activation_height = 0; // No activation delay
 
-        consensus.nMinimumChainWork = uint256{"00000000000000000000000000000000000000000000005faa15d02e6202f3ba"};
-        consensus.defaultAssumeValid = uint256{"000000005be348057db991fa5d89fe7c4695b667cfb311391a8db374b6f681fd"}; // 39550
+        consensus.nMinimumChainWork = uint256{"0000000000000000000000000000000000000000000001d455d7eda6382f7054"};
+        consensus.defaultAssumeValid = uint256{"00000000798bd720800263b69bee01aae57ea376596415a04423da29f000c367"}; // 72513
```

Comment:
In 92a6f7b238cd8ff015e242d6a515452c401133a7 "[kernel] update assumevalid and minimumChainWork for v29"

This block appears to have been reorged out. Perhaps we should choose a block with actual work for testnet4.

Reply:
Oops I shouldn't have just picked the tip. Now using 72600 which was mined 8 hours ago.